keyboards/handwired/practice60/bluepill/rules.mk

```
@@ -0,0 +1,43 @@
+# project specific files
+SRC =	matrix.c \
```

```suggestion
SRC =	\
```

Since I have no way to multi-line edit this here ....